pkg/operator/status.go

```diff
@@ -289,9 +299,175 @@ func (optr *Operator) syncUpgradeableStatus() error {
 		coStatus.Message = "One or more machine config pools are updating, please see `oc get mcp` for further details"
 	}
 
+	// don't overwrite status if updating or degraded
+	if !updating && !degraded {
+		skewStatus, status, err := optr.isKubeletSkewSupported(pools)
+		if err != nil {
+			glog.Errorf("Error checking version skew: %v, kubelet skew status: %v, status reason: %v, status message: %v", err, skewStatus, status.Reason, status.Message)
```

Comment:
Wheather this `glog.Errorf` casued the CI failures? https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_machine-config-operator/2658/pull-ci-openshift-machine-config-operator-master-okd-e2e-aws/1419855633997369344/artifacts/e2e-aws/gather-extra/artifacts/pods/openshift-machine-config-operator_machine-config-operator-598979445f-sd8qf_machine-config-operator.log